### PR TITLE
feat: arbitrum one

### PR DESCRIPTION
**Pull request by alfetopito**
# Description

Add Arbitrum one

# Changes

Update every place I could find when the networks are mentioned

- [x] reference/contracts/core
- [x] reference/contracts/periphery 
- [x] reference/core/auctions/rewards
- [x] reference/core/signing schemes
- [x] concepts/order types/twap orders
- [x] tutorials/cow swap/custom links
- [x] tutorials/cow swap/native
- [x] tutorials/cow swap/twap
- [x] tutorials/solvers/local test
- [x] tutorials/widget
- [x] tutorials/governance/token

### CODE REVIEW

File: docs/cow-protocol/reference/contracts/core/README.mdx [hunk 1]
```diff
@@ -42,9 +42,9 @@ Take care when [signing](../core/signing-schemes), ensuring that the `EIP-712` d
...
+| [`GPv2VaultRelayer`](core/vault-relayer)<br />Upgradeable: No ❎<br />[GitHub](https://github.com/cowprotocol/contracts/blob/main/src/contracts/GPv2VaultRelayer.sol) | `0xC92E8bdf79f0507f65a392b0ab4667716BFE0110`<br />- [Ethereum mainnet](https://etherscan.io/address/0xC92E8bdf79f0507f65a392b0ab4667716BFE0110)<br />- [Gnosis chain](https://gnosisscan.io/address/0xC92E8bdf79f0507f65a392b0ab4667716BFE0110)<br/>- [Arbitrum one](https://arbiscan.io/address/0xC92E8bdf79f0507f65a392b0ab4667716BFE0110)<br/>- [Sepolia](https://sepolia.etherscan.io/address/0xC92E8bdf79f0507f65a392b0ab4667716BFE0110) |
```
Comment (anxolin) [inline docs/cow-protocol/reference/contracts/core/README.mdx]: nit: should we provide the link to the `code` tab?

**Review by anxolin — APPROVED** [review]
Great! remember to link this PR to the process on listing new chains

File: docs/cow-protocol/reference/contracts/core/README.mdx (hunk 1 above)
Comment (mfw78) [reply docs/cow-protocol/reference/contracts/core/README.mdx]: Yeah, in hindsight, the `#code` tab would be nice :pray:

File: docs/governance/token.md [hunk 2]
```diff
@@ -10,17 +10,19 @@ At the core of CoW Protocol lies the COW token, which serves as a governance tok
...
 #### `COW` - Main token
 
-| **Chain** | **COW Token Address** |
-| --- | --- |
-| Ethereum | [`0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB`](https://etherscan.io/token/0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB) |
+| **Chain**    | **COW Token Address**                                                                                                  |
+|--------------|------------------------------------------------------------------------------------------------------------------------|
+| Ethereum     | [`0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB`](https://etherscan.io/token/0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB)  |
 | Gnosis Chain | [`0x177127622c4a00f3d409b75571e12cb3c8973d3c`](https://gnosisscan.io/token/0x177127622c4a00f3d409b75571e12cb3c8973d3c) |
+| Arbitrum One | [`0xcb8b5cd20bdcaea9a010ac1f8d835824f5c87a04`](https://arbiscan.io/token/0xcb8b5cd20bdcaea9a010ac1f8d835824f5c87a04) |
```
Comment (mfw78) [inline docs/governance/token.md]: nit: I think there should be some note here that this contract is _not_ deployed / developed by CoW, but however is the bridged version of the Ethereum Mainnet canonical token.

**Review by mfw78 — APPROVED** [review]

**Comment by alfetopito** [comment]
> Great! remember to link this PR to the process on listing new chains

https://www.notion.so/cownation/Update-docs-fde472e02d154c93b2f3ede0ac81f587?pvs=4

File: docs/governance/token.md (hunk 2 above)
Comment (alfetopito) [reply docs/governance/token.md]: That would apply to the one on Gnosis Chain as well then, right?

Comment (alfetopito) [reply docs/governance/token.md]: https://github.com/cowprotocol/docs/pull/390/commits/59c80a44cd9b36c15125dea369f02744fee9055f

File: docs/cow-protocol/reference/contracts/core/README.mdx (hunk 1 above)
Comment (alfetopito) [reply docs/cow-protocol/reference/contracts/core/README.mdx]: https://github.com/cowprotocol/docs/pull/390/commits/fcb497100e93f895969dd73023b5113b6168b327

File: docs/governance/token.md (hunk 2 above)
Comment (alfetopito) [reply docs/governance/token.md]: Doesn't look that good 
image

Any tips for doing this sort of thing? Couldn't find how to do foot notes

File: docs/governance/token.md [hunk 3]
```diff
@@ -10,17 +10,26 @@ At the core of CoW Protocol lies the COW token, which serves as a governance tok
...
+| **Chain**    | **COW Token Address**                                                                                                     |
+|--------------|---------------------------------------------------------------------------------------------------------------------------|
+| Ethereum     | [`0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB`](https://etherscan.io/token/0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB)     |
+| Gnosis Chain | [`0x177127622c4a00f3d409b75571e12cb3c8973d3c`](https://gnosisscan.io/token/0x177127622c4a00f3d409b75571e12cb3c8973d3c)`*` |
+| Arbitrum One | [`0xcb8b5cd20bdcaea9a010ac1f8d835824f5c87a04`](https://arbiscan.io/token/0xcb8b5cd20bdcaea9a010ac1f8d835824f5c87a04)`*`   |
+
+:::note
+
+`*` These contracts were not developed nor deployed by COW DAO, however, they are the bridged versions of the canonical token from Ethereum, using the official bridges.
+[Ommni bridge](https://gnosisscan.io/address/0xf6A78083ca3e2a662D6dd1703c939c8aCE2e268d#code) for Gnosis Chain and [Arbitrum One bridge](https://arbiscan.io/address/0x09e9222e96e7b4ae2a407b98d48e330053351eee#code) for Arbitrum one.
```
Comment (mfw78) [inline docs/governance/token.md]:
```suggestion
[Omni bridge](https://gnosisscan.io/address/0xf6A78083ca3e2a662D6dd1703c939c8aCE2e268d#code) for Gnosis Chain and [Arbitrum One bridge](https://arbiscan.io/address/0x09e9222e96e7b4ae2a407b98d48e330053351eee#code) for Arbitrum one.
```

File: docs/governance/token.md [hunk 4]
```diff
@@ -10,17 +10,26 @@ At the core of CoW Protocol lies the COW token, which serves as a governance tok
...
-| Gnosis Chain | [`0x177127622c4a00f3d409b75571e12cb3c8973d3c`](https://gnosisscan.io/token/0x177127622c4a00f3d409b75571e12cb3c8973d3c) |
+| **Chain**    | **COW Token Address**                                                                                                     |
+|--------------|---------------------------------------------------------------------------------------------------------------------------|
+| Ethereum     | [`0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB`](https://etherscan.io/token/0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB)     |
+| Gnosis Chain | [`0x177127622c4a00f3d409b75571e12cb3c8973d3c`](https://gnosisscan.io/token/0x177127622c4a00f3d409b75571e12cb3c8973d3c)`*` |
+| Arbitrum One | [`0xcb8b5cd20bdcaea9a010ac1f8d835824f5c87a04`](https://arbiscan.io/token/0xcb8b5cd20bdcaea9a010ac1f8d835824f5c87a04)`*`   |
+
+:::note
+
+`*` These contracts were not developed nor deployed by COW DAO, however, they are the bridged versions of the canonical token from Ethereum, using the official bridges.
```
Comment (mfw78) [inline docs/governance/token.md]:
```suggestion
`*` These contracts were not developed nor deployed by CoW DAO, however, they are the bridged versions of the canonical token from Ethereum, using the official bridges.
```

**Review by mfw78 — APPROVED** [review]
FYI, for footnotes: https://www.markdownguide.org/extended-syntax/#footnotes

File: docs/governance/token.md (hunk 2 above)
Comment (alfetopito) [reply docs/governance/token.md]: https://github.com/cowprotocol/docs/pull/390/commits/b2f4b2591164a0f07c1de7ec8e89817027866065

File: docs/cow-protocol/tutorials/cow-swap/native.mdx [hunk 5]
```diff
@@ -8,7 +8,7 @@ import CancellationNotGuaranteed from '../../../partials/_cancellation_not_guara
 
 # Native tokens
 
-In blockchain networks the "native token" - for example, Ether (ETH) on Ethereum & xDAI (xDAI) on Gnosis Chain - is the primary digital currency. 
+In blockchain networks the "native token" - for example, Ether (ETH) on Ethereum and Arbitrum One & xDAI (xDAI) on Gnosis Chain - is the primary digital currency.
```
Comment (MartinquaXD) [inline docs/cow-protocol/tutorials/cow-swap/native.mdx:11]: Is it expected to use `and` for an enumeration within a group and `&` to enumerate groups like this?

**Review by MartinquaXD — COMMENTED** [review]
Only checked the existing changes so far. Will check the docs as a whole again to see if there are more places that need to be changed.

File: docs/cow-protocol/tutorials/cow-swap/native.mdx (hunk 5 above)
Comment (alfetopito) [reply docs/cow-protocol/tutorials/cow-swap/native.mdx:11]: I defer this one to @mfw78

Comment (mfw78) [reply docs/cow-protocol/tutorials/cow-swap/native.mdx:11]: In lieu of hearing back from @gogonimago / @marshymarsh to the contrary, I don't consider this a blocker

**Review by MartinquaXD — APPROVED** [review]

**Review by shoom3301 — APPROVED** [review]

**Pull request merged**